### PR TITLE
IOW-113 - populate monitoring location table

### PR DESCRIPTION
**Pull request by dsteinich**
Note the coalesce whenever concatenating in sql - otherwise a null will result in the entire string being null.
Also, all trimming of the sitefile values is in the getters for the property.

### CODE REVIEW

File: src/test/resources/db/changelog/mysql/tables/qwResult/changeLog.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      id: "create.table.qw_result"
+      preConditions:
+        - onFail: MARK_RAN
+        - onError: HALT
+        - not:
+          - tableExists:
+              tableName: qw_result
+      changes:
+        - sqlFile:
+            path: qwResult.sql
```
Comment (mbucknell) [inline src/test/resources/db/changelog/mysql/tables/qwResult/changeLog.yml]: I don't see this file anywhere. What am I missing?

File: src/test/resources/db/changelog/mysql/tables/qwSample/changeLog.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      id: "create.table.qw_sample"
+      preConditions:
+        - onFail: MARK_RAN
+        - onError: HALT
+        - not:
+          - tableExists:
+              tableName: qw_sample
+      changes:
+        - sqlFile:
+            path: qwSample.sql
```
Comment (mbucknell) [inline src/test/resources/db/changelog/mysql/tables/qwSample/changeLog.yml]: Same question as above.

File: src/test/resources/db/changelog/mysql/tables/qwResult/changeLog.yml (hunk 1 above)
Comment (dsteinich) [reply src/test/resources/db/changelog/mysql/tables/qwResult/changeLog.yml]: This file (src/test/resources/db/changelog/mysql/tables/qwResult/changeLog.yml) should not have been committed yet. src/test/resources/db/changelog/mysql/tables/changeLog.yml does not reference it at this time. It is in the test section and I would like to leave it here for future reference.

File: src/test/resources/db/changelog/mysql/tables/qwSample/changeLog.yml (hunk 2 above)
Comment (dsteinich) [reply src/test/resources/db/changelog/mysql/tables/qwSample/changeLog.yml]: This file (src/test/resources/db/changelog/mysql/tables/qwSample/changeLog.yml) should not have been committed yet. src/test/resources/db/changelog/mysql/tables/changeLog.yml does not reference it at this time. It is in the test section and I would like to leave it here for future reference.

**Pull request merged**